app/views/users/_user.html.haml

```diff
@@ -1,4 +1,32 @@
-%tr
-  %td= link_to user.login, user
-  %td= l(user.created_at, :format => :short)
-  %td= l(user.updated_at, :format => :short)
+      %tr
```

Comment:
haml does not like the indenting at the start of the document. That's also why some of the tests are failing now.

Please make sure to start with no indenting:
```haml
%tr
  %td= link_to user.login, user
  %td= ...
```